File: sample-java/src/main/java/omnifit/sdk/brain/sample/SampleApplication.java
package omnifit.sdk.brain.sample;

import android.app.Application;
import android.content.Context;
import omnifit.sdk.brain.OmnifitBrain;
import timber.log.Timber;

public class SampleApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Timber.plant(new Timber.DebugTree());
        OmnifitBrain.init(this);
    }
}

File: sample-kotlin/src/main/java/omnifit/sdk/brain/sample/SampleApplication.kt
package omnifit.sdk.brain.sample

import android.app.Application
import omnifit.sdk.brain.OmnifitBrain
import timber.log.Timber

class SampleApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        Timber.plant(Timber.DebugTree())
        OmnifitBrain.init(this)
    }
}